Code/Editor/WelcomeScreen/WelcomeScreenDialog.cpp

```diff
@@ -162,6 +162,24 @@ bool WelcomeScreenDialog::eventFilter(QObject *watched, QEvent *event)
     return QDialog::eventFilter(watched, event);
 }
 
+bool WelcomeScreenDialog::IsValidLevelName(const QString& path)
+{
+    QStringList pathParts = Path::SplitIntoSegments(path);
+
+    QString levelName = pathParts.at(pathParts.size() - 1);
+    if (!levelName.endsWith(".prefab", Qt::CaseInsensitive))
+    {
+        return true;
+    }
```

Comment:
Is this meant to return `false` if it doesn't end in `.prefab` ?

Also, this will need to account for if prefabs are enable/disabled. We still need to allow `.ly` legacy level names if prefabs are disabled.

Reply:
I was just skipping the check for non prefab levels, but it makes sense to put that in as well, I'll get that done.

Reply:
@cgalvan Could you check you're happy with this change when you get a minute please?